pkgs/development/libraries/libnut/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchgit }:
```

Comment:
Usually people put a space here, or even better:

```suggestion
{ stdenv
, lib
, fetchgit
}:
```

Reply:
Done! Thank you for the suggestion.